app/routes/events/components/EventDetail/index.js

```diff
@@ -460,21 +476,6 @@ export default class EventDetail extends Component<Props, State> {
               </Flex>
             </ContentSidebar>
           </ContentSection>
-          <Link
-            to="/pages/arrangementer/26-arrangementsregler"
-            style={{ marginTop: 0 }}
-          >
-            <Flex alignItems="center">
-              <Icon name="document" style={{ marginRight: '4px' }} />
```

Comment:
> Can we also keep the icon? I like icons :sweat_smile:

I'll try to make it work. So far the formatting of this thing is what I've been struggling the most with, as the inconsistency you've mentioned was a bit too much with the text for allergies combined with the event-rules still having the icon..  
Might work better if some more of the inconsistencies are worked out though